apps/public-docsite-v9/src/Concepts/Poster/OverviewCard.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+
+import { Image, makeStyles, shorthands, tokens, typographyStyles } from '@fluentui/react-components';
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import logoImageSrc from '../../../public/fluent9.png';
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import chevronsImageSrc from '../../../public/fluent9-chevrons.png';
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import starsImageSrc from '../../../public/fluent9-stars.png';
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import springImageSrc from '../../../public/fluent9-spring.png';
+
+const useStyles = makeStyles({
+  root: {
+    ...shorthands.border(tokens.strokeWidthThick, 'solid', tokens.colorNeutralBackground3),
+    ...shorthands.borderRadius('3px'),
+    ...shorthands.overflow('visible'),
+    ...shorthands.padding('40px'),
+    boxSizing: 'border-box',
+    boxShadow: tokens.shadow16,
+    ...shorthands.margin('50px'),
+    display: 'flex',
+    flexDirection: 'column',
+  },
+  header: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+  },
+  title: {
+    ...typographyStyles.title1,
+  },
+  tagline: {
+    ...typographyStyles.caption1,
+  },
+  features: {
+    display: 'flex',
+    alignItems: 'center',
+    alignContent: 'center',
+  },
+  logo: {
+    alignSelf: 'center',
+    width: '400px',
+  },
+  featuresList: {
+    display: 'grid',
+    gridTemplateRows: '50px auto 50px auto 50px auto',
+    gridTemplateColumns: '180px',
+    alignItems: 'center',
+    justifyItems: 'center',
+    justifyContent: 'center',
+    rowGap: '5px',
+    ...shorthands.padding('30px', 0, 0, '30px'),
+  },
+  featureText: {
+    ...typographyStyles.caption2,
+    ...shorthands.margin('10px'),
+  },
+});
+
+export const OverviewCard: React.FunctionComponent = () => {
+  const styles = useStyles();
+
+  return (
+    <div className={styles.root}>
+      <div className={styles.header}>
+        <div className={styles.title}>Fluent UI React 9.0.0</div>
```

Comment:
Is it possible to pull the version number from `package.json` so for any given release this will reflect the precise version?

Reply:
Removed specific version, will let storybook handle that.  Sometimes the public website has a version dropdown too.